website/content/docs/installation.md

```diff
@@ -87,6 +87,15 @@ To get the command line completions with Bash, add the following to your `~/.bas
 . $HOME/.bloop/bash/bloop
 ```
 
+#### Fish completions
+
+To get the command line completions with fish shell, navigate to your local completion location (e.g.: `~/.config/fish/completions`)
+and create a symlink to the file provided in the Bloop distribution:
```

Comment:
Shall we mention here what to do if the bloop server is not running and how to get the fish completions to work after the user manually starts the server?

Reply:
Agree, makes sense to do so.

Reply:
Great, whenever you make the change, please rebase the PR so that i can merge it right away.

Reply:
Change done, and rebased against master.